src/sage/schemes/elliptic_curves/padic_lseries.py

```diff
@@ -61,7 +61,7 @@
 #                  https://www.gnu.org/licenses/
 ######################################################################
 
-import sage.matrix.all as matrix
+from sage.matrix.constructor import Matrix as matrix
 import sage.schemes.hyperelliptic_curves.monsky_washnitzer
```

Comment:
why not just `from sage.matrix.constructor import matrix`?

Reply:
OK, done (in all places)